react-basic-practice/src/App.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { ChakraProvider } from "@chakra-ui/react";
 import theme from "./themes";
-
+import Button from './components/Button';
```

Comment:
Please add more config for alias path

Reply:
https://blog.logrocket.com/using-path-aliases-cleaner-react-typescript-imports/

Reply:
i done